src/lib/objects/payggy/chat.svelte

```diff
@@ -36,8 +25,7 @@
 			console.error(res.error)
 		}
 	}
-	$: myMessage = data?.from === args.profile.address
-	// FIXME: will not work for group chats
+	$: myMessage = message.fromAddress === args.profile.address
 	$: otherUser = args.users.find((u) => u.address !== args.profile.address)
```

Comment:
```suggestion
	$: myMessage = message.fromAddress === args.profile.address
	// FIXME: will not work for group chats
	$: otherUser = args.users.find((u) => u.address !== args.profile.address)
```
I'd prefer to keep the note